fil_fungible_token/src/token/mod.rs

```diff
@@ -57,6 +59,8 @@ where
     msg: MSG,
     /// In-memory cache of the state tree
     state: TokenState,
+    /// ActorID
```

Comment:
Receiving actor ID, i.e. `message().receiver()`

But also why not make this a function of MSG? It's also an abstraction from the syscall interface